js/AttributeEditor.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+'use strict';
+
+/** Editor for attributes. **/
+class AttributeEditor {
+  onApply: Function;
+  operations: Array;
+
+  constructor(onApply: Function) {
+      this.onApply = onApply;
+      this.operations = [];
+  }
+
+  /**
+   * Adds string attribute.
+   * @instance
+   * @memberof AttributeEditor
+   * @function setString
+   *
+   * @param {string} value The attribute value.
+   * @param {string} name The attribute name.
+   * @return {AttributeEditor} The attribute editor instance.
+   */
+  setString(value: string, name: String): AttributeEditor {
```

Comment:
I think we can get away with this taking a String, Boolean, or Number in the future under a single `set` call